themes/finna2/templates/Related/RecordDriverRelatedRecordList.phtml

```diff
@@ -0,0 +1,23 @@
+<?php if ($data = $this->results): ?>
```

Comment:
Ota tähän `if (!($data = $this->results) { return: }` niin ei tartte koko template olla if:n sisällä (tai korjata sisennystä).

Reply:
Tämä tiedosto on väärässä PR:ssä, päivitin korjauksen tänne:
https://github.com/NatLibFi/NDL-VuFind2/pull/1699/files#diff-2f446288c6eca35de2c026e769f629f7R1